lib/resolvers/functions/creatorCheck.js

```diff
@@ -1,12 +1,13 @@
 const { UnauthorizedError } = require('errors');
 const requestContext = require('talawa-request-context');
+const { USER_NOT_AUTHORIZED } = require('../../../constants');
 
 const creatorCheck = (context, org) => {
   const isCreator = String(org.creator) === context.userId;
   if (!isCreator) {
     throw new UnauthorizedError(
       process.env.NODE_ENV !== 'production'
```

Comment:
hey @ebrahim354, you're almost there. You can also place the `'production'` string in the constants file as well